apps/ex_nvr_web/lib/ex_nvr_web/live/device_list_live.ex

```diff
@@ -92,6 +101,40 @@ defmodule ExNVRWeb.DeviceListLive do
             </div>
           </.button>
         </:action>
+        <:action :let={device}>
+          <.modal id={"delete-device-modal-#{device.id}"}>
+            <div class="bg-white dark:bg-gray-800 m-8 rounded">
+              <h2 class="text-xl text-white font-bold mb-4">
+                Are you sure you want to delete this device? <br />
+              </h2>
+              <h3>
+                The actual recording files are not deleted. <br />
+                If you want to delete them delete the following folders: <br />
+                <div class="bg-gray-400 rounded-md p-4 mt-2">
+                  <code class="text-gray-800">
+                    <%= Device.recording_dir(device) %> <br />
```

Comment:
If you want to suggest the path to delete, just suggest: `Device.base_dir(device)` is enough.